apps/web/src/components/(dashboard)/layout/profile-dropdown.tsx

```diff
@@ -35,8 +37,8 @@ export type ProfileDropdownProps = {
 
 export const ProfileDropdown = ({ user }: ProfileDropdownProps) => {
   const { theme, setTheme } = useTheme();
-
   const { getFlag } = useFeatureFlags();
+  const userIsAdmin = isAdmin(user);
```

Comment:
Silly rule, but I'd be consistent with the below flag using the `is<resource>` naming scheme so `userIsAdmin => isUserAdmin` 😄